dashboard/test/models/user_test.rb

```diff
@@ -1066,7 +1069,10 @@ def assert_fails_email_uniqueness_validation(user)
     level_group = create :level_group, name: 'LevelGroupLevel1', type: 'LevelGroup'
     level_group.properties['pages'] = [{levels: ['level_multi1', 'level_multi2']}]
```

Comment:
Maybe an existing bug in this test: What are `level_multi1` and `level_multi2`? Are these supposed to be the level names `sublevel1` and `sublevel2`?

Reply:
Great question - I don't know. I'm still fuzzy on level groups.